tensorflow/core/kernels/mkl_lrn_op.cc

```diff
@@ -0,0 +1,775 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
We used LRN Python test for testing the kernel. Graph rewrite pass rewrites them into our LRN kernel.